io.catenax.mandatory_dismantling/1.0.0/MandatoryDismantling.ttl

```diff
@@ -0,0 +1,991 @@
+#######################################################################
+# Copyright (c) 2024 SAP SE
+# Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft
+# Copyright (c) 2024 LRP Autorecycling Leipzig GmbH
+# Copyright (c) 2024 Fraunhofer-Gesellschaft zur F�rderung der angewandten Forschung e.V.
+# Copyright (c) 2024 ZF Friedrichshafen AG
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.mandatory_dismantling:1.0.0#> .
+@prefix ext-number: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix ext-quantity: <urn:samm:io.catenax.shared.quantity:2.0.0#> .
+@prefix ext-uuid: <urn:samm:io.catenax.shared.uuid:2.0.0#> .
+
+:MandatoryDismantling a samm:Aspect ;
+   samm:preferredName "MandatoryDismantling"@en ;
+   samm:description "The mandatory dismantling data model is intended to ensure that end-of-life vehicles are disposed of properly. It defines components, parts and materials that must be mandatorily dismantled, as well as the necessary provision of information on those components, parts and materials to be dismantled"@en ;
+   samm:properties ( [ samm:property :catenaXId; samm:optional true ] :van :bbpNumberDismantler :substanceOfConcern :depollutionRequirement :depollutionRecord :digitallyCodedComponent :mandatoryRemoval ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:catenaXId a samm:Property ;
+   samm:preferredName "Catena-X ID"@en ;
+   samm:description "The fully anonymous Catena-X ID of the serialized part, valid for the Catena-X dataspace."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "urn:uuid:580d3adf-1981-44a0-a214-13d6ceed9379" .
+
+:van a samm:Property ;
+   samm:preferredName "VAN"@en ;
+   samm:description "Vehicle Anonymiyed Number. This should be deducted from a reference to the vehicle TWIN"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "WBA41DU060S228332" .
+
+:bbpNumberDismantler a samm:Property ;
```

Comment:
is this a typo and you mean bpNumberDismanter = bpnDIsmanter? What is bbp?

Reply:
Yes, that's a typo. bpNumber Dismantler mean bpnDismantler